src/Baka/Traits/KanvasModelTrait.php

```diff
@@ -77,6 +77,15 @@ public static function getById(mixed $id, ?AppInterface $app = null): self
         }
     }
 
+    public static function getByStripeId(string $stripeId): self
```

Comment:
@Keysie27 do we need thi sin the trait on all method or just a few? , since you re using it on anther trait, you can do 1 trait and just include it where needed